### PR TITLE
Fix docs for running tests

### PR DESCRIPTION
**Pull request by brandonchinn178**
The instructions didn't seem to work for me. Let me know if these aren't the right instructions either.

### CODE REVIEW

File: cabal-testsuite/README.md [hunk 1]
```diff
@@ -4,11 +4,12 @@ frameworks.
 How to run
 ----------
 
-1. Build `cabal-tests` (`cabal build cabal-tests`)
+1. Build `cabal-testsuite` (`cabal build cabal-testsuite`)
```
Comment (ffaf1) [inline cabal-testsuite/README.md:7]: Mhhh I wonder if this maybe could be slightly more future-proof/correct as `cabal build cabal-testsuite:cabal-tests`)?

File: cabal-testsuite/README.md [hunk 2]
```diff
@@ -4,11 +4,12 @@ frameworks.
...
 2. Run the `cabal-tests` executable. It will scan for all tests
    in your current directory and subdirectories and run them.
-   To run a specific set of tests, use `cabal-tests PATH ...`.  You can
-   control parallelism using the `-j` flag.
+   To run a specific set of tests, use `cabal-tests PATH ...`.
+   (e.g. `cabal exec -- cabal-tests cabal-testsuite/PackageTests/BuildTargets/UseLocalPackage/use-local-version-of-package.ts.hs`)
```
Comment (Mikolaj) [inline cabal-testsuite/README.md:11]: The `exec` is to automatically find the full path of the exe? I wonder what's the best idiom for that. Another is `$(cabal list-bin exe:cabal-tests)` and I'm sure there are others. Which one works best for you? Doesn't `exec` also do some potentially disruptive things behind the scenes (I don't remember)?

Comment (brandonchinn178) [reply cabal-testsuite/README.md:11]: Yes, exec finds the path. I use Stack mostly; in Cabal projects, is it common for people to "just know" they have to find the test executable somehow?

Or maybe the docs should reference `cabal test` instead?

Comment (ulysses4ever) [reply cabal-testsuite/README.md:11]: `cabal run cabal-tests` worked fine for me. I assume `cabal test cabal-tests` would too. I don't see why we would advertise such a niche command as `cabal exec`.

Comment (brandonchinn178) [reply cabal-testsuite/README.md:11]: Sure, again, I don't use Cabal, so I didn't know what command to use. I don't particularly care which command to document, but we should document one of them

Comment (ulysses4ever) [reply cabal-testsuite/README.md:11]: Fully agree. Thanks a lot for taking the time to prepare a patch!

File: cabal-testsuite/README.md (hunk 1 above)
Comment (brandonchinn178) [reply cabal-testsuite/README.md]: Fixed: f972cf89339d2265676e32c4628975c24b63f726

File: cabal-testsuite/README.md (hunk 2 above)
Comment (brandonchinn178) [reply cabal-testsuite/README.md]: Fixed: f972cf89339d2265676e32c4628975c24b63f726

**Comment by Mikolaj** [comment]
CI was borked. Let me rebase to include a fix to CI.

**Comment by Mikolaj** [comment]
@mergify rebase

**Comment by Mikolaj** [comment]
@brandonchinn178: I've set the needs-review label. Is that fine with you?

@ulysses4ever, @ffaf1: any more comments from you?

**Review by ulysses4ever — APPROVED** [review]
LGTM, thanks!

**Review by tchoutri — APPROVED** [review]
LGTM!

**Comment by ulysses4ever** [comment]
@brandonchinn178 feel free to apply one of the merge-me labels to hand it over to the bot…

**Pull request merged**